inst/templates/get-started.Rmd

```diff
@@ -383,6 +383,49 @@ vroom::vroom_write(to_edit, "to_edit.csv", delim = ",")
 writexl::write_xlsx(to_edit, "to_edit.xlsx")
 ```
 
+### Calculate coverage statistics
+
+```{r}
+# Matched companies based on suggested matches
+matched_rows <- to_edit %>%
+  filter(suggest_match) %>%
+  nrow()
+
+# Share of matched companies from total loanbook companies
+matched_share <- to_edit %>%
+  filter(suggest_match) %>%
+  nrow() / loanbook %>%
```

Comment:
Mmm .. this looks strange:

```r
  nrow() / loanbook %>%
```

I'll run it to see what it actually does.

Reply:
@maurolepore I am not sure how to resolve this yet. So please guide me if you know the right way

Reply:
@maurolepore I am not sure how to resolve this yet. So please guide me if you know the right way